.gitignore

```diff
@@ -34,3 +34,5 @@ build/
 *-local.yml
 
 /temp/
+
+*my.conf
```

Comment:
I see that spring uses `local` for naming that should be ignored. Let's follow that convention and rename your custom NGINX config file to `local.conf` and update this line accordingly.